renderer/components/result/MeasurementRow.js

```diff
@@ -158,18 +139,17 @@ const StatusBox = ({testName, testKeys, isAnomaly}) => {
 // XXX still need to show the summary in here
 const TestRow =  ({measurement, query, testKeys, isAnomaly}) => {
 
-  const icon = iconMap[measurement.test_name]
-  let fullnameID = fullnameMap[measurement.test_name]
-  if (!fullnameID) {
-    fullnameID = 'Test.MISSING.Fullname'
-  }
+  const icon = tests[measurement.test_name].icon && React.cloneElement(
```

Comment:
Maybe we want this to not fail if the `test_name` is missing?

Reply:
This will most likely happen when we try to use `$OONI_HOME` which was created/modified by a `probe-cli` version not supported by `probe-desktop`. If `test_name` is missing, quite a few things will break across the app. If we have to handle all such cases, I suggest we do that in a separate PR.

Reply:
It could be as simple as no loading the measurement for unsupported `test_name`s

Reply:
Opened https://github.com/ooni/probe/issues/986